File: RB101_Programming_Foundations/small_problems/Easy2/Run2/how_big_is_the_room.rb
SQMETERS_TO_SQFEET = 10.7639
puts "Enter the length of the rooom in meters:"
length = gets.chomp.to_f
puts "Enter the width of the room in meters:"
width = gets.chomp.to_f

area_m = length * width
area_sqf = area_m * SQMETERS_TO_SQFEET
puts "The area of the room is %0.1f square meters (%0.2f square feet)." % [area_m, area_sqf]

File: js210_fundamentals_of_JS_Programming/problem_sets/easy1/how_big_is_the_room.js
const readlineSync = require('readline-sync');

const SQMETERS_TO_SQFEET = 10.7639;
console.log('Enter the length of the room in meters:');
let length = readlineSync.prompt();
length = parseInt(length, 10);
console.log('Enter the width of the room in meters:');
let width = readlineSync.prompt();
width = parseInt(width, 10);
const areaSqM = length * width;
const areaSqF = areaSqM * SQMETERS_TO_SQFEET;

console.log(`The area of the room is ${areaSqM.toFixed(2)} square meters (${areaSqF.toFixed(2)} square feet).`);
